benchexec/tools/ultimate.py

```diff
@@ -445,14 +442,14 @@ def _determine_result_with_propertyfile(
                 return status
         return result.RESULT_UNKNOWN
 
-    def get_value_from_output(self, lines, identifier):
-        # search for the text in output and get its value,
-        # stop after the first line, that contains the searched text
-        for line in lines:
-            if identifier in line:
-                start_position = line.find("=") + 1
-                return line[start_position:].strip()
-        return None
+    def get_value_from_output(self, output, identifier):
+        regex = re.compile(identifier)
+        for line in output:
+            match = regex.search(line)
+            if match and len(match.groups()) > 0:
+                return match.group(1)
+        logging.debug("Did not find a match with regex %s", identifier)
+        return "N/A"
```

Comment:
`None` is in general a better return value. Using a string will lead to table columns with mixed numbers and strings, so there won't be any statistics etc. With `None` the respective cells will be correctly interpreted as "missing value".